app/packages/multimodal/src/adapters/mcap/react/McapLoadingAscii.module.css

```
@@ -0,0 +1,41 @@
+  display: inline-block;
+  flex: 0 0 auto;
+  height: 22px;
+  overflow: hidden;
+  position: relative;
+  width: 28px;
+}
+
+.fallingAsciiBoxInner {
+  animation: mcapAsciiFall 1.15s ease-in infinite;
+  display: block;
+  font-family: "SFMono-Regular", Consolas, "Liberation Mono", monospace;
```

Stylelint (17.12.0)

[error] Expected no quotes around "SFMono-Regular" (font-family-name-quotes)

(font-family-name-quotes)